client/ayon_shotgrid/addon.py

```diff
@@ -33,26 +40,13 @@ def get_plugin_paths(self):
     def create_shotgrid_session(self):
         from .lib import credentials
 
-        sg_username, sg_password = credentials.get_local_login()
-
-        if not sg_username or not sg_password:
-            return None
+        sg_username = os.getenv("AYON_SG_USERNAME")
```

Comment:
Where do you normally define this environment variable, and what should happen if it's not set empty?

Reply:
on my case I'm actually just using `$USER` as all our artists machines have that defined so we don't need any extra user input to define their login for SG but that's why I left it open to discuss. We could potentially expose an option on the settings where admins can choose any existing env var to use as SG login and/or optionally expose the login menu so each user can set it themselves on the registry